docs/UserGuide.md

```diff
@@ -522,6 +522,13 @@ from John Doe's appointment.
 | List Archived | `apmt alist`                                          | -                               |
 | Sort          | `apmt sort`                                           | -                               |
 
+#### Prescription-related Commands
```

Comment:
I'm not sure if putting it under a separate Prescription-related commands is better or to nest it under `Appointment Commands`, since anw, that is how it is structured in code.

Perhaps we could add a line `and all prescription commands` under `Appointment Commands`

Reply:
Edited

Reply:
thank you